### ISSUE TITLE
Crash while running the pub

### ISSUE THREAD

**Issue by humble-git** [YehudaKremer/msix#20]
Hi,
I'm trying to run the package with default application but i'm getting following error 

```
Option /o specified, existing files will be overwritten.
Using default hash method: SHA256.
The path (/p) parameter is: "\\?\C:\Users\Humble\Projects\github_desktop\build\windows\runner\Release\github_desktop.msix"
The content directory (/d) parameter is: "\\?\C:\Users\Humble\Projects\github_desktop\build\windows\runner\Release"
Enumerating files from directory "\\?\C:\Users\Humble\Projects\github_desktop\build\windows\runner\Release"
Packing 16 file(s) in "\\?\C:\Users\Humble\Projects\github_desktop\build\windows\runner\Release" (content directory) to "\\?\C:\Users\Humble\Projects\github_desktop\build\windows\runner\Release\github_desktop.msix" (output file name).
Memory limit defaulting to 12818999296 bytes.
Using "\\?\C:\Users\Humble\Projects\github_desktop\build\windows\runner\Release\AppxManifest.xml" as the manifest for the package.
MakeAppx : error: Failure at appxFactory->CreateManifestReader(manifestStream, &manifestReader) - 0x80080204 - The specified package format is not valid: The package manifest is not valid.
MakeAppx : error: Error info: /*[local-name()="Package" and namespace-uri()="http://schemas.microsoft.com/appx/manifest/foundation/windows10"]/*[local-name()="Identity" and namespace-uri()="http://schemas.microsoft.com/appx/manifest/foundation/windows10"][1]/@Name
'com.flutter.github_desktop' violates pattern constraint of '[-.A-Za-z0-9]+'.
The attribute 'Name' with value 'com.flutter.github_desktop' failed to parse.
Cleaning up output file "\\?\C:\Users\Humble\Projects\github_desktop\build\windows\runner\Release\github_desktop.msix".
MakeAppx : error: Failure at (CreatePackage( overwrite, hashAlgorithm, fileList, outputPath, manifestStream.Get(), forceCompressionNone, performanceOptions, encryptPackage, encryptionOptions, cgmPath, mainPackagePathForResourceExemption, makepriExeFullPath)) - 0x80080204 - The specified package format is not valid: The package manifest is not valid.
MakeAppx : error: Package creation failed.
MakeAppx : error: 0x80080204 - The specified package format is not valid: The package manifest is not valid.
```

**Comment by YehudaKremer** [comment]
Hello
your "**identity_name**" is "**com.flutter.github_desktop**", this configuration field cannot have "**_**" in it.

please change it by adding this to your pubspec.yaml file:
```
msix_config:
  identity_name: com.flutter.githubdesktop
```

**Comment by YehudaKremer** [comment]
Hello @humble-git this bug fixed, see: #25 
please use version **0.1.15**
